deploy.sh

```diff
@@ -2,6 +2,8 @@
 set -e
 
 stack_name="$1"
+common_stack_name="${2:-kbv-common-cri-api-local}"
+secret_prefix="${3:-kbv-cri-api}"
```

Comment:
we should make the `common_stack_name` and `secret_prefix` optional parameters that default to the values on line 26 and 27 if they are not specified
In a similar manner to:
https://github.com/alphagov/di-ipv-cri-common-lambdas/blob/d5a5eeb8fb3557edfd9dc40989063056c75ce4b6/deploy.sh#L23